### ISSUE TITLE
FF Type-0 Graphical bug (Android) 

### ISSUE THREAD

**Issue by Saramagrean** [hrydgard/ppsspp#11631]
### What happens?

Look like https://github.com/hrydgard/ppsspp/issues/9926 but it happens every time, When entering or ending the battle scene.

Just in Android use backend OpenGL only, on Vulkan is look fine. and for Windows not get this issue. 

OpenGL
![screenshot_20181203-204201](https://user-images.githubusercontent.com/26355472/49377898-e9a88000-f73d-11e8-9ab2-6339296cfe71.png)
![screenshot_20181203-204231](https://user-images.githubusercontent.com/26355472/49377954-0e045c80-f73e-11e8-940f-332d8955dc50.png)

### What *should* happen?

Vulkan
![screenshot_20181203-203858](https://user-images.githubusercontent.com/26355472/49378014-38561a00-f73e-11e8-89e9-f41965833496.png)
![screenshot_20181203-203817](https://user-images.githubusercontent.com/26355472/49378124-6c313f80-f73e-11e8-8158-e7191c839f41.png)

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.

PPSSPP latest git build (1.7.1-241) use default settings.

Huawei P9+ (Kirin 955, Mali-T880)

• GE Dump from OpenGL
[NPJH50443.ppdmp.zip](https://github.com/hrydgard/ppsspp/files/2639735/NPJH50443.ppdmp.zip)

**Comment by hrydgard** [comment]
Just noting that it looks a little similar to https://github.com/hrydgard/ppsspp/issues/9926 which you also reported...

**Comment by Saramagrean** [comment]
@hrydgard Roger. (•̀ᴗ•́)و

@unknownbrackets Last work: v1.1.1
First break: maybe v1.1.1-202 (I have only this version, can't find v1.1.1 build between 1-201) 

v1.1.1
![screenshot_20181204-111137](https://user-images.githubusercontent.com/26355472/49418713-257f2c00-f7b6-11e8-95f1-43648c3497a9.png)

v1.1.1-87
![screenshot_20181204-111236](https://user-images.githubusercontent.com/26355472/49418734-3cbe1980-f7b6-11e8-8cec-c82d42fd3f46.png)

v1.1.1-202
![screenshot_20181204-111514](https://user-images.githubusercontent.com/26355472/49418752-4fd0e980-f7b6-11e8-9402-871372b577e3.png)

**Comment by unknownbrackets** [comment]
Assuming that's v1.1.1-202-g067431b and v1.1.1-87-g488aec1.  But the tricky thing is other things could've been fixed or broken different ways since then since it was so many commits ago... hopefully not.

Here's that commit range:
https://github.com/hrydgard/ppsspp/compare/v1.1.1-87-g488aec1...v1.1.1-202-g067431b

There are many possible causes hiding in there...

-[Unknown]

**Comment by hrydgard** [comment]
Yeah, this has to have been a driver bug with vertex formats. Without repro and not a game-breaker, not really worth keeping open indeed.
